java/client/src/org/openqa/selenium/remote/BrowserType.java

```diff
@@ -27,11 +27,11 @@
   String GOOGLECHROME = "googlechrome";
   String SAFARI = "safari";
   /**
-   * @deprecated Use OPERA_BLINK
+   * @deprecated Use {@link #OPERA}
```

Comment:
Wouldn't this `@deprecated` comment go above `OPERA_BLINK` instead of `OPERA`?

Reply:
Maybe my IDE is playing tricks on me, I am sure the comment is above `OPERA_BLINK`. 
Please recheck, if this is still the case then I resend a commit.

This is what my IDE says:
<img width="461" alt="image" src="https://user-images.githubusercontent.com/7461869/82651769-83a30780-9c14-11ea-94ae-1e3e2a372ef2.png">